configure-buildfarm/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+
+-
+  name: "Include release specific image versions"
+  include_vars: ../shared_files/{{ rhmap_version }}.yml
+
```

Comment:
Not sure how this will work in the community version? They wont have this yaml file